app/src/main/res/layout-land/resume_lesson_fragment.xml

```diff
@@ -68,7 +68,11 @@
         <Button
           android:id="@+id/resume_lesson_start_over_button"
           style="@style/StartOverLessonButton"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
           android:layout_marginEnd="12dp"
+          android:minWidth="128dp"
```

Comment:
This logic of min width is not the best solution.

Reply:
Updated